### PR TITLE
feat(Manager): Adding create symlink support from manager

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. Added a form to create a symlink from manager
2. Showing list of symlinks created in separate section in settings tab

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+export const schema = yup.object({
+  // TODO: change this to URL validation, after server supports http protocol append
+
+  source: yup
+    .string()
+    .label('Source Directory')
+    .trim()
+    .required()
+    .test('source', 'Source directory entered is incorrect', (value) =>
+      /^[_.a-zA-Z0-9/-]+$/.test(value?.toString() || '')
```
Comment (soumyadip007) [inline packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx:28]: `/^[_.a-zA-Z0-9/-]+$/` store it in a constant

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    .test('target', 'Target directory entered is incorrect', (value) =>
+      /^[_.a-zA-Z0-9/-]+$/.test(value?.toString() || '')
+    ),
+  env: yup
+    .string()
+    .label('Environment Name')
+    .trim()
+    .noWhitespace()
+    .max(15)
+    .matches(/^[a-zA-Z0-9-]+$/, 'Only letters, numbers, and dashes are allowed')
```
Comment (soumyadip007) [inline packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx:44]: `const folderValidation = /^[a-zA-Z0-9-]+$/`
store it in a constant and reuse it

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    .test('source', 'Source directory entered is incorrect', (value) =>
+      /^[_.a-zA-Z0-9/-]+$/.test(value?.toString() || '')
+    ),
+  target: yup
+    .string()
+    .label('Target Directory')
+    .trim()
+    .required()
+    .test('target', 'Target directory entered is incorrect', (value) =>
+      /^[_.a-zA-Z0-9/-]+$/.test(value?.toString() || '')
```
Comment (soumyadip007) [inline packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx:36]: `const folderValidation = /^[a-zA-Z0-9-]+$/`
store it in a constant and reuse it

**Review by soumyadip007 — CHANGES_REQUESTED** [review]
@nikhitamore please go through the chanegs

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx]: done

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx]: done

File: packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx (hunk 3 above)
Comment (nikhitamore) [reply packages/manager/src/views/Settings/components/Symlink/CreateSymlink/CreateSymlink.tsx]: done

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Pull request merged**